api/app/signals/settings/base.py

```diff
@@ -254,6 +254,7 @@
 CELERY_EMAIL_CHUNK_SIZE = 1
 CELERY_RESULT_BACKEND = 'django-db'
 CELERY_TASK_RESULT_EXPIRES = 604800  # 7 days in seconds (7*24*60*60)
+CELERY_TASK_ALWAYS_EAGER = os.getenv('CELERY_TASK_ALWAYS_EAGER', 'False') in TRUE_VALUES
```

Comment:
Just wondering why you added this here.

Reply:
Part of my local development workflow.

Reply:
I think that in general we need to take a look at the setting files and make them more consistent. Not blocking for this PR.